integration/test/ecs/ecs_metadata/ecs_metadata_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package ecs_metadata
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/amazon-cloudwatch-agent/integration/test"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs"
+)
+
+// Purpose: Detect the changes in metadata endpoint for ECS Container Agent https://github.com/aws/amazon-cloudwatch-agent/blob/master/translator/util/ecsutil/ecsutil.go#L67-L75
+// Implementation: Checking if a log group's the format(https://github.com/aws/amazon-cloudwatch-agent/blob/master/translator/translate/logs/metrics_collected/prometheus/ruleLogGroupName.go#L33) 
+// exists or not  since the log group's format has the scrapping cluster name from metadata endpoint.
+
+const (
+	RetryTime             = 10
+	// Log group format: https://github.com/aws/amazon-cloudwatch-agent/blob/master/translator/translate/logs/metrics_collected/prometheus/ruleLogGroupName.go#L33
+	ECSLogGroupNameFormat = "/aws/ecs/containerinsights/%s/prometheus" 
+)
+
+var clusterName = flag.String("clusterName", "", "Please provide the os preference, valid value: windows/linux.")
+
+func TestNumberMetricDimension(t *testing.T) {
+	// test for cloud watch metrics
+	ctx := context.Background()
+	client := test.GetCWLogsClient(ctx)
+
+	for currentRetry := 1; ; currentRetry++ {
+		if currentRetry == RetryTime {
+			t.Fatalf("Test metadata has exhausted %v retry time", RetryTime)
+		}
+		describeLogGroupInput := cloudwatchlogs.DescribeLogGroupsInput{
+			LogGroupNamePrefix: aws.String(fmt.Sprintf(ECSLogGroupNameFormat, *clusterName)),
+		}
+		describeLogGroupOutput, err := client.DescribeLogGroups(ctx, &describeLogGroupInput)
+
+		if err != nil {
+			t.Errorf("Error getting log group data %v", err)
+		}
+
+		if len(describeLogGroupOutput.LogGroups) > 0 {
+			break
+		}
```

Comment:
So all we want to verify is that the log group gets created? how do we ensure that this is unique across integration test runs?

Reply:
We will create a `unique random hex` every time [a test run](https://github.com/aws/amazon-cloudwatch-agent/pull/458/files/90951f325cfe1a47d9192747e42f25c580c6e011#diff-5cf3a68c413f5f031a2713f7488ab0c4668f7a4895120e1dfba706c50b28bba6R6-R8). Therefore, ensuring the test is unique across the integration test